resources/configurationFiles/analysisRNAseqGRCh38.xml

```diff
@@ -0,0 +1,52 @@
+
+        <cvalue name='STAR_PARAMS_CHIMERIC' value='--chimSegmentMin 15 --chimScoreMin 1 --chimScoreJunctionNonGTAG 0 --chimJunctionOverhangMin 15  --chimSegmentReadGapMax 3 --alignSJstitchMismatchNmax 5 -1 5 5 --chimOutType Junctions SeparateSAMold' type='string'/>
+        <!-- Hg38 reference genome -->
+        <cvalue name='hg38BaseDirectory' value='/icgc/ngs_share/assemblies/hg_GRCh38' type='path'/>
+        <cvalue name='hg38DatabaseDirectory' value='${hg38BaseDirectory}/databases' type='string'/>
+        <cvalue name='hg38IndexDirectory' value='${hg38BaseDirectory}/indexes' type='string'/>
+
+        <cvalue name='databaseDirectory' value='${hg38DatabaseDirectory}' type='string'/>
+        <cvalue name='indexDirectory' value='${hg38IndexDirectory}' type='string'/>
+
+        <!--Databases and index-->
+        <cvalue name='GENOME_FA' value='${hg38BaseDirectory}/sequence/GRCh38_decoy_ebv_phiX_alt_hla_chr.fa' type='string'/>
+        <cvalue name='GENOME_GATK_INDEX' value='${hg38IndexDirectory}/bwa/bwa07_GRCh38_decoy_ebv_phiX_alt_hla_chr/GRCh38_decoy_ebv_phiX_alt_hla_chr.fa' type='string'/>
+
+        <cvalue name='GENE_MODELS' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.gtf' type='string'/>
+        <cvalue name='GENE_MODELS_NOGENE' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.nogene.gtf' type='string' 
+                description="Like GENE_MODELS, but without 'gene' features. Required for gencode update for hg38. Defaults (for hg19): $GENE_MODELS."/>
+        
+        <cvalue name='GENE_MODELS_EXCLUDE' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.chrXYMT.rRNA.gtf' type='string'
+                description="Other than mitochondrial genes, there were no genes annotated as tRNA, so have removed it from the filename."/>
+        <cvalue name='GENE_MODELS_DEXSEQ' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.dexseq.gff' type='string'/>
+        
+        <cvalue name='GENOME_STAR_INDEX_50' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_50bp' type='string'/>
+        <cvalue name='GENOME_STAR_INDEX_100' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_100bp' type='string'/>
+        <cvalue name='GENOME_STAR_INDEX_200' value='${hg38IndexDirectory}/STAR/STAR_2.7.6a_Gencode31/STAR_2.7.6a_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_200bp' type='string'/>
+        <cvalue name='GENOME_KALLISTO_INDEX' value='${hg38IndexDirectory}/kallisto/kallisto-0.46.0_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31_k31/kallisto-0.46.0_GRCh38_decoy_ebv_phiX_alt_hla_chr_Gencode31.noGenes.index' type='string'/>
+        <cvalue name='ARRIBA_KNOWN_FUSIONS' value='${hg38BaseDirectory}/tools_data2/arriba/known_fusions_hg38_GRCh38_v2.0.0.tsv.gz' type='string'/>
```

Comment:
I have created the `tools_data2` in `ngs_share` due to write permission issues with `tools_data` folder, now this is fixed. So I will move the Arriba files into `tools_data`

Reply:
`v2.0.0` is the Arriba version. There is no equivalent for hg38 for version prior to 2.0.0. I can build one if you want.

Reply:
If it works, its fine. This is the hg38 configuration and everything is fine there, so don't bother. I just wanted to make sure this was not overlooked but **not** working.

Reply:
Yes, known fusion database file was missing for hg38 in v1.2.0. If v2.0.0 database files are OK here, I would leave it as well.

Reply:
I am not sure if this would work. At best, it will throw a whole lot of warnings and not actually have any effect. In the worst case, it would crash. What we can certainly do is to put an empty dummy file there. But I can also build a file for hg38. It's not too much work.

Reply:
@suhrig The v2.0.0 hg38 files works with the Arriba v1.2.0 and didn't throw any errors. If it is not too much work, could you please generate these files for the lower Arriba version?

Reply:
I can confirm there are no errors/crashes, but a whole lot of warnings. In fact, the entire content of the file is ignored, so there is no point in supplying it. I will build a file for version 1.2.0 later today.

Reply:
Thanks for the files @suhrig I have updated them here.